yellowbrick/features/manifold.py

```diff
@@ -514,49 +471,75 @@ def manifold_embedding(
         ``"tsne"``     `t-SNE <http://scikit-learn.org/stable/modules/manifold.html#t-distributed-stochastic-neighbor-embedding-t-sne>`_
         ============== ==========================
 
-    n_neighbors : int, default: 10
+    n_neighbors : int, default: None
```

Comment:
Thank you for making sure the quick method is consistent! With the docstring changes from above would you mind also ensuring that they're made down here as well!

Reply:
Sure!